code/game/objects/structures/props.dm

```diff
@@ -883,6 +883,16 @@
 	layer = ABOVE_MOB_LAYER
 	density = FALSE
 
+/obj/structure/prop/invuln/leaky_pipe
+	name = "pipe water"
+	icon = 'icons/obj/structures/props/watercloset.dmi'
+	icon_state = "water"
+	desc = "The Almayer has sprung a leak!"
+	density = FALSE
+
+/obj/structure/prop/invuln/leaky_pipe/colony
```

Comment:
you can be smart and make it get this description on initialize

this is pseudocode
```dm
/Initialize(mapload)
 . = ..()
 if(ismainshiplevel(z))
  desc = "[MAINSHIPNAME] has sprung a leak!"
 return
 if(isgroundlevel(z))
  desc = "[SSmapping.configs[GROUND_MAP].map_name] has sprung a leak!"
```